src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect, useMemo, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable';
+import { useSearchParams } from 'react-router-dom';
+import {
+  filterPersonByCenturies,
+  filterPersonByQuery,
+  getPeopleWithParents,
+  getSortPeople,
+} from '../utils/functions';
+import { SortBy } from '../types/SortBy';
+import { PeopleFilters } from '../components/PeopleFilters';
+
+export const PeoplePage: React.FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [isError, setIsError] = useState('');
```

Comment:
if you store error here do not use `is` in the start. Use `is`, `has` for boolean variables
 ```suggestion
  const [error, setError] = useState('');
```